dashboards/sap-hana/README.md

```diff
@@ -1,11 +1,18 @@
 ### Dashboards for SAP HANA
 
-#### Notes
+|SAP HANA GCE Overview|
+|:------------------|
+|Filename: [sap-hana-gce-overview.json](sap-hana-gce-overview.json)|
+|This dashboard has charts displaying `Instance Memory`, `Service Memory Used`, `Schema Estimated Max Memory`, `Rowstore Memory Size`, `Schema Read Outliers`,`Schema Write Outlines`, `Schema Record Count Outliers`, `System Connections`, `Alerts`, and `Blocked Transactions` from SAP HANA as well as charts of infrastructure related metrics for the running SAP HANA VMs: `CPU % Top 5 VMs`, `Memory % Top 5 VMs`, and `Hosts by Region`. There is also a card with links to docs and the SAP HANA logs in Cloud Logging.|
 
+#### Notes
```

Comment:
Please move the notes header back to the top of the file. You can remove the notes content with the suggested description change above.

Reply:
done